### PR TITLE
Add hostname/dns support for the module online_status

### PR DESCRIPTION
**Pull request by ritze**
Now you can check the online status via ping.

### CODE REVIEW

**Comment by ultrabug** [comment]
Thanks @ritze ! wow that's a chain of PRs ;)

I'd rather you use python [subprocess call](https://docs.python.org/2/library/subprocess.html#subprocess.call) please ?

File: py3status/modules/online_status.py [hunk 1]
```diff
@@ -25,18 +27,24 @@ class Py3status:
...
+    format_online = u'●'
     timeout = 2
     url = 'http://www.google.com'
 
     def _connection_present(self):
-        try:
-            urlopen(self.url, timeout=self.timeout)
-        except:
-            return False
+        if re.search('://', self.url):
```
Comment (tobes) [inline py3status/modules/online_status.py]: `if '://' in self.url:` would save having to use the re module

**Comment by ritze** [comment]
Is this the right way to push the stdout and stderr to /dev/null?

File: py3status/modules/online_status.py [hunk 2]
```diff
@@ -33,18 +33,17 @@ class Py3status:
...
             else:
                 return True
         else:
-            if os.system("ping -c 1 " + self.url + ">> /dev/null 2>&1") == 0:
-                return True
-            else:
-                return False
+            fnull = open(os.devnull, 'w')
+            return subprocess.call(['ping', '-c', '1', self.url], stdout=fnull,
+                               stderr=subprocess.STDOUT) == 0
```
Comment (tobes) [inline py3status/modules/online_status.py]: use `stderr=fnull` too

**Comment by ultrabug** [comment]
Thanks mate, good addition !

**Pull request merged**